code/modules/mob/inventory.dm

```diff
@@ -291,7 +291,7 @@
 	return !!locate(/obj/item) in held_items
 
 /mob/proc/drop_all_held_items()
-	. = FALSE
+	. = list()
```

Comment:
You need to check all consumers of this proc because this return value has changed

Though at a glance it seems only 1 use in the eyestab element is affected by this

Reply:
I didand fixced it